src/commands/flutter-symbols/upload.ts

```diff
@@ -345,9 +345,14 @@ export class UploadCommand extends Command {
       retries: 5,
       useGzip: true,
     })
-    this.context.stdout.write(`Mapping upload finished: ${result}\n`)
 
-    return result
+    if (status === UploadStatus.Success) {
+      this.context.stdout.write('Mapping upload finished\n')
+    } else {
+      this.context.stdout.write(`Mapping upload failed\n`)
```

Comment:
Does the error code get output in `verbose`? If not, can we output the error code here? Just thinking that if someone has a problem the extra information would be nice.

Reply:
The error code can only be `0`, `1` or `2`:
https://github.com/DataDog/datadog-ci/blob/50a2a70b23bad6e36ddc6d9f54bb886f44a4058e/src/helpers/upload.ts#L49-L53

And in this case ([`uploadMultipartHelper()`](https://github.com/DataDog/datadog-ci/blob/50a2a70b23bad6e36ddc6d9f54bb886f44a4058e/src/commands/flutter-symbols/helpers.ts#L20-L24) directly returns [`upload()`](https://github.com/DataDog/datadog-ci/blob/50a2a70b23bad6e36ddc6d9f54bb886f44a4058e/src/helpers/upload.ts#L59-L86)'s return value), we never possibly return a `UploadStatus.Skipped`, so I figured "Mapping upload finished" or "Mapping upload failed" was enough.

WDYT @fuzzybinary?

Reply:
Yeah sounds good. Thanks for taking care of this 👍